projects/demo/src/components/appearance/index.html

```diff
@@ -12,6 +12,7 @@
     </td>
 </tr>
 <tr
+    *ngIf="tuiDocAppearance && !hiddenOptions.includes('appearance')"
```

Comment:
No need to check `tuiDocAppearance`. When it's empty it means we need to show all appearances (stored in `appearances`)

Reply:
https://github.com/taiga-family/taiga-ui/pull/9067#discussion_r1766579682
`In appearance let's hide it if we passed empty array` 
But I don't understand you..

Reply:
You added `hiddenOptions` array, it is more flexible than my original suggestion. Just remove `tuiDocAppearance && ` from `*ngIf` and it will be fine

Reply:
nice, removed